Source/Core/Core/IOS/ES/Identity.cpp

```diff
@@ -19,14 +19,22 @@ namespace HLE
 {
 namespace Device
 {
-IPCCommandResult ES::GetConsoleID(const IOCtlVRequest& request)
+ReturnCode ES::GetDeviceId(u32* console_id) const
```

Comment:
You're right, this may actually fail (in IOS, it just calls IOSC_GetData and uses that as the return code). I'll fix this.